src/main/java/org/venice/beachfront/bfapi/model/Scene.java

```diff
@@ -34,10 +34,11 @@
 @Entity
 @Table(name = "__beachfront__scene")
 public class Scene {
-	public static final String PLATFORM_PLANETSCOPE = "planetscope";
-	public static final String PLATFORM_RAPIDEYE = "rapideye";
-	public static final String PLATFORM_SENTINEL = "sentinel";
-	public static final String PLATFORM_LANDSAT = "landsat";
+	public static final String PLATFORM_PLANET_PLANETSCOPE = "planetscope";
+	public static final String PLATFORM_PLANET_RAPIDEYE = "rapideye";
+	public static final String PLATFORM_PLANET_SENTINEL = "sentinel";
+	public static final String PLATFORM_PLANET_LANDSAT = "landsat";
+	public static final String PLATFORM_LOCALINDEX_LANDSAT = "landsat_pds";
```

Comment:
landsat vs. landsat_pds is also a bit ambiguous.  If we end up looking purely at the DB tables, for all of these Jobs, it's not going to be clear at all between "landsat" and "landsat_pds".  Maybe "landsat_bf" or something?

Reply:
"PDS" stands for "Public Dataset", so i think `landsat_pds` is an apt name for where it comes from. I'd actually have liked to rename `landsat` to `landsat_planet` but figured that would cause unnecessary confusion and test breakage, so I left it as is.